src/core/linter-rules/check-punctuation.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Linter rule "check-punctuation". Makes sure the there are no punctuations missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "check-punctuation";
+
```

Comment:
here, let's add:

```JS
const punctuationMarks = [".", ":", "!", "?"];
```

That will allow us to more easily manage these throughout the code.